.github/workflows/playwright.yml

```diff
@@ -0,0 +1,31 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ master, setup-redesign ]
+  pull_request:
+    branches: [ master, setup-redesign ]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: lts/*
+    - name: Install dependencies
+      run: npm ci
+    - name: Install backend dependencies
+      run: |
+        cd backend-mock
+        npm ci
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+    - name: Run Playwright tests
+      run: npx playwright test
+    - uses: actions/upload-artifact@v4
+      if: always()
```

Comment:
In the documentation this line is specified with `${{ !cancelled() }}`.

https://playwright.dev/docs/ci-intro#on-pushpull_request

Reply:
This was created by playwright itself during the setup... Will update